acceptance-tests/src/main/java/io/blueocean/ath/ATHJUnitRunner.java

```diff
@@ -1,11 +1,10 @@
 package io.blueocean.ath;
 
-import com.google.common.collect.Lists;
-import com.google.common.io.Files;
 import com.google.inject.Guice;
 import com.google.inject.Injector;
 import io.blueocean.ath.pages.classic.LoginPage;
 import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
`lang3` has caused me some classloading issues in the past. Can't remember what it was though so might not be an issue any more.